src/lib/coda_base/transaction_witness.ml

```diff
@@ -18,4 +14,5 @@ module Stable = struct
   module Latest = V1
 end
 
-type t = Stable.Latest.t = {ledger: Sparse_ledger.t} [@@deriving sexp]
+type t = Stable.Latest.t = {ledger: Sparse_ledger.Stable.Latest.t}
```

Comment:
I think the field should have type `Sparse_ledger.Stable.V1.t`, because that's what's in `Stable.Latest.t`.